cs-commons/src/main/java/io/cloudslang/content/constants/ExceptionsValues.java

```diff
@@ -0,0 +1,15 @@
+package io.cloudslang.content.constants;
+
+/**
+ * Created by victor on 31.08.2016.
+ */
+public class ExceptionsValues {
+    public static final String EXCEPTION_DELIMITER = ": ";
+    public static final String INVALID_BOOLEAN_VALUE = "The provided boolean string is invalid. It must be 'true' or 'false'";
+    public static final String INVALID_INTEGER_VALUE = "The provided integer string is invalid.";
+    public static final String INVALID_DOUBLE_VALUE = "The provided double string is invalid.";
+    public static final String INVALID_BOUNDS = "The lower bound has to be less than the upper bound";
```

Comment:
I can also suggest considering `need to be`, `it is required`, `it is expected to be` :bowtie: